docs/get-started/sidebars.ts

```diff
@@ -85,7 +85,11 @@ module.exports = {
       ],
     },
     'research-papers',
-    'glossary',
+    {
+      type: 'link',
+      href: '/learn/glossary',
```

Comment:
Do we want to link to glossary here?
I would either:
- Have its own page
- Remove the glossary in one of the sections

I guess I prefer the later

Reply:
Let's remove it then :)